design/pedibus/override/templates/avaiability/avaiability.tpl

```diff
@@ -0,0 +1,190 @@
+{set-block scope=root variable=cache_ttl}0{/set-block}
+{def $__AVAIABILITIES = fetch(
+                                'content',
+                                'list',
+                                hash(
+                                    'parent_node_id', $node.node_id,
+                                    'class_identifier', 'disponibilita'
+                                )
+                        )
+}
+
+{if $__AVAIABILITIES|count()}
+    {def $current_user = fetch( 'user', 'current_user' )}
+    {def $__LINES_FILTER = array()}
+    {if $current_user.is_logged_in}
+        {if $current_user.contentobject.class_identifier|eq("user_pedibus")}
+            {foreach $current_user.contentobject.data_map.lines_enable_to_read.content.relation_list as $__LINE_REF}
+                {set $__LINES_FILTER = $__LINES_FILTER|append($__LINE_REF.node_id)}
+            {/foreach}
+        {/if}
+    {/if}
+
+{*    {$__LINES_FILTER|attribute(show)}*}
+
+    {def $__DAY_IN_SECOND = ezini( 'IlPedibus', 'seconds_on_day', 'ilpedibus.ini' )}
+    {def $__DAYS_MAP = ezini( 'IlPedibus', 'days_map', 'ilpedibus.ini' )}
+    {literal}
+    <style>
+
+    body {
+        margin: 40px 10px;
+        padding: 0;
+        font-family: "Lucida Grande",Helvetica,Arial,Verdana,sans-serif;
+        font-size: 14px;
+    }
+
+    #pedibuscalendar
+    {
+        max-width: 900px;
+        margin: 0 auto;
+        padding: 0 10px;
+    }
+
+    </style>
+    {/literal}
+
+
+
+    <script type="text/javascript">
+    {literal}
+        document.addEventListener("DOMContentLoaded", function()
+        {
+            (function ($)
+            {
+                $(document).ready(function ()
+                {
+                    $('#pedibuscalendar').fullCalendar(
```

Comment:
usare il fullcalendar senza chiamate asincrone è inutile e oneroso
conviene usare il calendario di default di ez oppure creare un modulo ad hoc che popoli il full calendar via ajax

Reply:
Allora, si potrebbe si creare un modulo per chiamate ajax e usare un $.get o $.post o $.ajax di jquery per recuperare il json e poi popolare quello che sarà l'init del calendario.... Potrebbe essere un estensione aggiuntiva questa?
Avevamo un limite di budget = tempo: quindi ho cercato la soluzione che permettesse di lavorare in parallelo con la grafica che si voleva svliluppare.
Infatti il calendario di ez non permette di fare i "giochi" che sono stati fatti usando fullcalendar.io

Reply:
Sto facendo il modulo per caricare con Ajax i dati e poi popolare il calendario, in modo da non appesantire la visualizzazione della pagina